### PR TITLE
Trim requisition action

### PR DESCRIPTION
**Pull request by dannyroberts**
This is as far as I could easily get with https://trello.com/c/usgtYuDe/28-requisitionactions-and-related-code. My hunch is a lot of the code I `# todo`'d can just be removed, but I wanted to put this up for review and see what you thought.

@czue @gcapalbo

### CODE REVIEW

**Comment by gcapalbo** [comment]
:+1: looks ok to me so far, will defer to @czue on the todos

File: corehq/apps/commtrack/models.py [hunk 1]
```diff
@@ -193,6 +157,7 @@ class CommtrackConfig(QuickCachedDocumentMixin, Document):
     multiaction_enabled = BooleanProperty()
     multiaction_keyword_ = StringProperty()
 
+    # todo: remove?
     requisition_config = SchemaProperty(CommtrackRequisitionConfig)
     openlmis_config = SchemaProperty(OpenLMISConfig)
```
Comment (czue) [inline corehq/apps/commtrack/models.py]: (not part of this PR but it seems like `OpenLMISConfig` might also be able to be removed)

File: corehq/apps/commtrack/sms.py [hunk 2]
```diff
@@ -131,6 +131,7 @@ def parse(self, text):
             RequisitionActions.FULFILL,
             RequisitionActions.RECEIPTS
         ]:
+            # todo: soft assert? remove?
```
Comment (czue) [inline corehq/apps/commtrack/sms.py]: yeah i think this can all be removed (maybe we can add soft asserts along with the other sms functionality you're also removing)

File: corehq/apps/commtrack/sms.py [hunk 3]
```diff
@@ -409,9 +410,11 @@ def process_transactions(E, transactions):
         if tx.action in (
             const.StockActions.STOCKONHAND,
             const.StockActions.STOCKOUT,
+            # todo: remove?
```
Comment (czue) [inline corehq/apps/commtrack/sms.py]: :+1:

File: corehq/apps/commtrack/sms.py [hunk 4]
```diff
@@ -409,9 +410,11 @@ def process_transactions(E, transactions):
...
             const.RequisitionActions.REQUEST
         ):
             balances.append(tx)
+        # todo: remove?
```
Comment (czue) [inline corehq/apps/commtrack/sms.py]: :+1:

File: corehq/apps/commtrack/sms.py [hunk 5]
```diff
@@ -422,6 +425,7 @@ def process_transactions(E, transactions):
 
 
 def process_balances(E, balances):
+    # todo: remove checks for requisition actions
```
Comment (czue) [inline corehq/apps/commtrack/sms.py]: :+1:

File: corehq/apps/commtrack/sms.py [hunk 6]
```diff
@@ -586,6 +591,7 @@ def to_instance(data):
             *stock_blocks
         )
         return etree.tostring(root, encoding='utf-8', pretty_print=True)
+    # todo: remove?
```
Comment (czue) [inline corehq/apps/commtrack/sms.py]: :+1:

File: corehq/apps/commtrack/models.py [hunk 7]
```diff
@@ -98,34 +81,15 @@ def name(self):
     def is_stock(self):
         return self.action in STOCK_ACTION_ORDER
 
-    @property
-    def is_requisition(self):
-        return self.action in REQUISITION_ACTION_TYPES
-
 
+# todo: delete this?
```
Comment (czue) [inline corehq/apps/commtrack/models.py]: yep, I thinks so.

**Comment by czue** [comment]
hey, yeah I think pretty much everything you TODO'd can be removed. was thinking we could also add in soft asserts for the SMS processing just to make sure it's not in use like the other PR

**Comment by czue** [comment]
this is fun

File: corehq/apps/commtrack/models.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/commtrack/models.py]: Is OpenLMIS not a thing we need to support anymore? Just wanting to know
because I've come across other code that can be removed if that's the case

Comment (czue) [inline corehq/apps/commtrack/models.py]: yeah i think we can remove all of it. i thought it was already removed, but sounds like i didn't give danny complete instructions: https://trello.com/c/dcfnPhR0/14-openlmis-integration-code

Comment (dannyroberts) [inline corehq/apps/commtrack/models.py]: Ha! I removed a lot of stuff in https://github.com/dimagi/commcare-hq/pull/12414 but I guess there were other pockets not revealed by the things I was grepping for / the thread I was pulling on. I'll add this comment thread to that ticket

Comment (snopoke) [inline corehq/apps/commtrack/models.py]: The one I'd come across was v0.3 of ProductAPI:
https://github.com/dimagi/commcare-hq/pull/12787#discussion_r74426280

Comment (dannyroberts) [inline corehq/apps/commtrack/models.py]: Thanks I'll take a look at that too
